docs/devices/0402946.md

```diff
@@ -1,30 +1,48 @@
 ---
-title: "Datek 0402946 control via MQTT"
-description: "Integrate your Datek 0402946 via Zigbee2MQTT with whatever smart home
+title: "Zigbee module for ID LOCK 150 via MQTT"
+description: "Integrate your ID LOCK 150 via Zigbee2MQTT with whatever smart home
```

Comment:
This line is generated (VENDOR + MODEL ID) so please don't change it (will be reverted on the next release otherwise). Same for https://github.com/Koenkk/zigbee2mqtt.io/pull/769/files#diff-306651b0cef7f0f3470e6029441d7ff73b3e019f14d1dcded37a1b9e075cc747L10